src/main/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/dto/LocationStatus.kt

```diff
@@ -5,6 +5,22 @@ enum class LocationStatus(
 ) {
   ACTIVE("Active"),
   INACTIVE("Inactive"),
-  NON_RESIDENTIAL("Non-residential"),
   ARCHIVED("Archived"),
+  DRAFT("Draft"),
+}
+
+enum class DerivedLocationStatus(
+  val description: String,
+) {
+  ACTIVE("Active"),
+  INACTIVE("Inactive"),
+  ARCHIVED("Archived"),
+  DRAFT("Draft"),
```

Comment:
Draft being a status means that users will have to wait until after publishing to make the location non-res, are we ok with that?

Reply:
Don't know what they have designed about making a draft location convertible to a non-res.  For now I suspect they won't be creating non-res at this point in the process.  Potentially a draft cell could be made non-res as its not yet been published.  There are many scenarios to work through here - this is just the beginning